Create folder design with all mockups (#54)
KrispyTech/airneis: docs/design/README.md

# Figma Link

[Figma link](https://www.figma.com/file/tqpdNMq2MpwY135z4RKXi3/AIRNEIS?type=design&node-id=39%3A1014&mode=design&t=nh2gdlrOpHCehg5D-1)

# Brief Presentation

_In Figma project, the are three unique pages:_

## Templates

![Templates](./images/templates.png)

> A page containing all the templates we need to use the fonts, logo, colours and icons.

Reviewer: ```suggestion
A page containing all the templates we need to use:  fonts, logo, colors, and icons.
```

> This is the basis of the project, which introduces the main theme of our application and website.

Reviewer: What do you mean by this?

Author: This page contains all the design elements for our website.

## Components

![Components](./images/components.png)

A page containing all the components needed to create a page.

## Pages

![Pages](./images/pages.png)

> A page containing all the website and mobile mockups.

Reviewer: ```suggestion
A page containing the website and mobile mock-ups.
```

> A visual of each page in web and mobile are included.

Reviewer: ```suggestion
We provide a visual of each page for both displays: web & mobile
```
